instagramcrawler.py

```diff
@@ -323,12 +327,14 @@ def main():
                         help='Number of posts to download: integer')
     parser.add_argument('-c', '--caption', action='store_true',
                         help='Add this flag to download caption when downloading photos')
+    parser.add_argument('-l', '--headless', type=int, default=0,
```

Comment:
I'd recommend storing this in the same way the captions are stored using `action='store_true'`, that way you can just pass `-l` without having to supply `1`
```
parser.add_argument('-l', '--headless', action='store_true',
                                   help='If set, will use PhantomJS driver to run script as headless')
```